AdaptiveRoads/Manager/NetInfoExtension.cs

```diff
@@ -264,11 +264,8 @@ public class Net : ICloneable, ISerializable {
             public float OneOverSinOfParkingAngle = 1;
 
 #if QUAY_ROADS_SHOW
-            [CustomizableProperty("Quay Road", "Properties")]
+            public Data.QuayRoads.ProfileSection[] quayRoadsProfile = null;
```

Comment:
public members should be CamelCase (start with capital letter)

Reply:
resolved in next commit.